src/styles/main.scss

```
@@ -1,3 +1,16 @@
+@import "./utils/variables.scss";
+@import "./blocks/calendar.scss";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.container__content {
+  margin: 50px auto;
 }
```

that's only 3 rows, but since it (container) is a separate BEM block it should be in a separate file
```suggestion
.container__calendar {
  margin: 50px auto;
}
```